docs/verify/local-testing-mode.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Qiskit Runtime local testing mode
+description: Use local testing mode to simulate quantum hardware in Qiskit Runtime
+---
+
+<span id="test-locally"></span>
+# Qiskit Runtime local testing mode
+
+Quantum simulators should be used to help develop and test programs before fine-tuning them and sending them to real quantum hardware. Local simulators can do this with good performance and efficiency.  Clifford circuits can be simulated very efficiently, and results can be verified. This is useful to give confidence in an experiment.
```

Comment:
This first paragraph feels like it should go in the verify introduction page, as it is talking more broadly about simulation as a whole, not specifically local testing mode.